src/Gateways/https-wrapper.ts

```diff
@@ -27,7 +27,7 @@ export const request = (
       res.on("error", reject);
     });
     req.on("socket", (socket: Socket) => {
-      socket.setTimeout(this.timeout || 100000);
+      socket.setTimeout(100000);
```

Comment:
this was erroring out locally for me, it may be a npm link issue instead of something that needs to be changed